README.md

```diff
@@ -102,3 +113,49 @@ To use the seed tables which have some info around common types, run:
 ```sh
 dbt seed
 ```
+
+## Incremental modeling
```

Comment:
Do we need to discuss running the incremental updates?

Reply:
@camphillips22 it's the same command to run incremental updates as it would be to run dbt normally, just:
```
dbt run
```
It will definitely take _longer_ to run, so I can include that information!

Reply:
Right, but to get new data in the incremental tables, `dbt run` has to be run periodically. Coordinating that is more what I mean. Like DBT cloud or other solutions.